spec/features/login_visit_candidate_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+feature 'Visit Election' do
+  let(:user) { create(:user) }
+  let(:election) { create(:election,
+                         start: Time.zone.now-5.days,
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Surrounding space missing for operator '-'.